packages/pn-personagiuridica-webapp/src/pages/UnderConstruction.page.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { Box, Typography } from '@mui/material';
```

Comment:
If this page is not used any more, remove it

Reply:
it's used when clicking on the sidebar items "Utenti" and "Gruppi"